doc/api/http.md

```diff
@@ -1520,15 +1537,13 @@ added: v0.3.6
   * `headers` {Object} An object containing request headers.
   * `auth` {String} Basic authentication i.e. `'user:password'` to compute an
     Authorization header.
-  * `agent` {http.Agent|Boolean} Controls [`Agent`][] behavior. When an Agent
-    is used request will default to `Connection: keep-alive`. Possible values:
+  * `agent` {http.Agent|Boolean} Controls [`Agent`][] behavior. Possible values:
    * `undefined` (default): use [`http.globalAgent`][] for this host and port.
    * `Agent` object: explicitly use the passed in `Agent`.
-   * `false`: opts out of connection pooling with an Agent, defaults request to
-     `Connection: close`.
+   * `false`: causes a new `Agent` with default values to be used.
```

Comment:
Isn't `Connection: close` the default, since you have to explicitly opt-in via `keepAlive: true` when creating a new `Agent`? So if the defaults are used, then that means `keepAlive: false` is used ...